draft-ietf-core-groupcomm-bis.md

```diff
@@ -527,13 +529,15 @@ When securing CoAP group communication with Group OSCORE {{I-D.ietf-core-oscore-
 
 ### Client Handling of Multiple Responses With Same Token ### {#sec-request-response-multi}
-Since a client sending a group request with a Token T will accept multiple responses with the same Token T, it is possible in particular that the same server sends multiple responses with the same Token T back to the client. For example, this server might not implement the optional CoAP message deduplication based on Message ID; or it might be acting out of specification as a malicious, compromised or faulty server.
+Since a client sending a group request with a Token T will accept multiple responses with the same Token T, it is possible in particular that the same server sends multiple responses with the same Token T back to the client.
+
+For example, if the client sends a group request specifying the Observe option set to 0 (see {{Section 3.1 of RFC7641}}) and this server adds the client to the list of observers for the targeted resource, then the server will send multiple responses as Observe notifications to notify the client of changes to the resource state (see {{Section 4.2 of RFC7641}}). The use of Observe with group communication is discussed in more details in {{sec-observe}}. As an alternative example, this server might not implement the optional CoAP message deduplication based on Message ID; or it might be acting out of specification as a malicious, compromised or faulty server.
 
 When this happens, it is up to the specific client implementation to decide at which layer deduplication of responses is performed, or whether it is necessary in an application at all. If the processing of a response is successful, the client delivers the response to the application as usual.
 
-The application itself can be in a good position to decide what to do, depending on the available context information. For instance, it might accept and process all the responses from the same server, even if they are not Observe notifications (i.e., they do not include an Observe option). Alternatively, the application might accept and process only one of those responses, such as the most recent one from that server, e.g., when this can trigger a change of state within the application.
+The application itself can be in a good position to decide what to do, depending on the available context information. For instance, it might accept and process all the responses from the same server, even if they are not Observe notifications (i.e., they do not include an Observe option). Alternatively, the application might accept and process only one of those responses, such as the one that is deemed to be the most recent one from that server, e.g., when this can trigger a change of state within the application.
```

Comment:
My problem still is that, from a client's point of view, each incoming response message (unless detected to be reordered) will be deemed to be the most recent one from that server, so I don't understand how this statement is selective.

Reply:
Maybe it's just better to drop the central part of the sentence and say:

> Alternatively, the application might accept and process only one of those responses, e.g., when this can trigger a change of state within the application.

Reply:
Addressed in https://github.com/core-wg/groupcomm-bis/pull/44/commits/f58845b1f2f542506ddf1c6b8479748fa9599e07